examples/validating-public-input/contracts/script/Deployer.s.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.12;
+
+import {Script, console} from "forge-std/Script.sol";
+import {FibonacciValidator} from "../src/FibonacciValidator.sol";
+
+contract CounterScript is Script {
```

Comment:
Not a big deal but should rename to something more relevant like 
```suggestion
contract FibonacciDeployer is Script {
```

Reply:
Solved in [7bc5736](https://github.com/yetanotherco/aligned_layer/pull/648/commits/7bc5736e446209ff3da92b740696fea242c00b8c)